content/en/blog/2021-12-23-data-curation-baleen.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "Data Curation: A Whale of a Problem"
+slug: "data-curation-baleen"
+date: 2021-12-24T11:37:43-05:00
+draft: false
+image_webp: images/blog/whale.webp
+image: images/blog/whale.jpg
+author: Nabiha Naqvie
```

Comment:
I removed all the social media links here since that information is already stored [here](https://github.com/rotationalio/rotational.io/blob/main/content/en/author/nabiha-naqvie.md) -- hugo will parse the string on line 8 to lookup your author record and create the link automagically!